sensors/static/sensors/js/overview.js

```diff
@@ -0,0 +1,197 @@
+  type: "line",
+  data: {
+    datasets: [{
+      backgroundColor: soundGradient,
+      label: "Sound"
+    }]
+  },
+  options: {
+    scales: {
+      yAxes: [{
+        ticks: {
+          callback: function(value, index, values) {
+            return value + " dB";
+          }
+        }
+      }]
+    },
+    tooltips: {
+      callbacks: {
+        label: function(tooltipItem, data){
+          return "Volume: " + tooltipItem.yLabel.toFixed(2) + " dB"
+        }
+      }
+    }
+  }
+})
+const peopleChart = new Chart(peopleCtx, {
+  type: "line",
+  data: {
+    datasets: [{
+      backgroundColor: peopleGradient,
+      label: "People"
+    }]
+  },
+  options: {
+    scales: {
+      yAxes: [{
+        ticks: {
+          callback: function(value, index, values) {
+            return value + " people";
+          }
+        }
+      }]
+    },
+    tooltips: {
+      callbacks: {
+        label: function(tooltipItem, data){
+          return "Number of people: " + tooltipItem.yLabel.toFixed(0)
+        }
+      }
+    }
+  }
+})
+
+function getTemperatureData(){
+  return fetch(measurementUrl + `?type=temperature`)
+    .then(response => response.json())
+    .then(jsonData => {
+      const dataValues = jsonData.map(x => x.value)
+      const dataLabels = jsonData.map(x => moment(x.created_at).format("HH:mm"))
+      temperatureChart.data.labels = dataLabels
+      temperatureChart.data.datasets[0].data = dataValues
+      temperatureChart.update()
+    })
+}
+
+function getHumidityData(){
+  return fetch(measurementUrl + `?type=humidity`)
+    .then(response => response.json())
+    .then(jsonData => {
+      const dataValues = jsonData.map(x => x.value)
+      const dataLabels = jsonData.map(x => moment(x.created_at).format("HH:mm"))
+      humidityChart.data.labels = dataLabels
+      humidityChart.data.datasets[0].data = dataValues
+      humidityChart.update()
+    })
+}
+
+function getSoundData(){
+  return fetch(measurementUrl + `?type=sound`)
+    .then(response => response.json())
+    .then(jsonData => {
+      const dataValues = jsonData.map(x => x.value)
+      const dataLabels = jsonData.map(x => moment(x.created_at).format("HH:mm"))
+      soundChart.data.labels = dataLabels
+      soundChart.data.datasets[0].data = dataValues
+      soundChart.update()
+    })
+}
+
+function getPeopleData(){
+  return fetch(measurementUrl + `?type=people`)
+    .then(response => response.json())
+    .then(jsonData => {
+      const dataValues = jsonData.map(x => x.value)
+      const dataLabels = jsonData.map(x => moment(x.created_at).format("HH:mm"))
+      peopleChart.data.labels = dataLabels
+      peopleChart.data.datasets[0].data = dataValues
+      peopleChart.update()
+    })
+}
+
+getTemperatureData()
+getHumidityData()
+getSoundData()
+getPeopleData()
+
+setInterval(getTemperatureData, 20 * 1000)
+setInterval(getHumidityData, 20 * 1000)
+setInterval(getSoundData, 20 * 1000)
+setInterval(getPeopleData, 20 * 1000)
```

Comment:
This means the functions should be called every 20 seconds, right? Why not have this value defined a a constant?